go.mod

```diff
@@ -1 +1,3 @@
-module github.com/onrik/logrus
+module github.com/Decentr-net/logrus
```

Comment:
Why?

Reply:
I've started using. I'll recreate pull request without this commit